MetaMask/metamask-mobile
# Fix link to testnet faucets

## **Description**

Fix the link to testnet faucets.

## **Manual testing steps**

_1. Start a transaction and go to confirmation page (ensure that you have less that required ETH on the testnet)
_2. Click the **buy** option, it should load correct faucet page

## **Related issues**

_Fixes: MetaMask/mobile-planning#1255

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
